test/res.send.js

```diff
@@ -250,6 +250,70 @@ describe('res', function(){
     })
   })
 
+  describe('.send(Blob)', function(){
+    var buffer = require('buffer');
+
+    it('should send as blob type', function (done) {
+      if (buffer.Blob) {
```

Comment:
why these `if (buffer.Blob) ` checks in the tests ?
How could it be falsy ?

Reply:
> How could it be falsy ?

`Blob` is a *new* Node.js feature and will not exist in older versions.

> Added in: v15.7.0, v14.18.0

https://nodejs.org/dist/latest-v19.x/docs/api/buffer.html#class-blob

Reply:
Because blob is not there in every version of the node as I see the minimum version of the node is 0.10 so that's why i skiped these testcases against all those node versions.